frontend/public/components/pod-exec.jsx

```diff
@@ -39,14 +39,15 @@ const PodExec_ = connectToFlags(FLAGS.OPENSHIFT)(
       this.state = {
         open: false,
         containers: [],
-        activeContainer: _.get(props, 'obj.spec.containers[0].name'),
+        activeContainer: props.activeContainer
+          ? props.activeContainer
```

Comment:
I would change this prop name to `initialContainer` to make it clear that it's only used on initialization.